elliottlib/cli/find_builds_cli.py

```diff
@@ -159,6 +153,25 @@ def _json_dump(as_json, unshipped_builds, kind, tag_pv_map):
                 json.dump(json_data, json_file, indent=4, sort_keys=True)
 
 
+def _fetch_builds_by_id(builds, tag_pv_map):
+    green_prefix('Build NVRs provided: ')
+    click.echo('Manually verifying the builds exist')
+    builds_tuple = []
+    for tag in tag_pv_map:
+        latest_builds = brew.get_latest_builds('nvr', tag, builds)
+        builds_tuple.extend(_gen_nvrp_tuples(latest_builds, tag_pv_map, tag))
```

Comment:
`get_latest_builds` only accepts a list of component names rather than build NVRs or Ids. See https://github.com/openshift/elliott/pull/113